src/main/frontend/handler/editor.cljs

```diff
@@ -3148,16 +3148,7 @@
     (when-let [block-id (:block/uuid current-block)]
       (if (= format "embed")
        (copy-block-ref! block-id #(str "{{embed ((" % "))}}"))
-       (copy-block-ref! block-id block-ref/->block-ref))
-      (notification/show!
-       [:div
-        [:span.mb-1.5 (str "Block " format " copied!")]
```

Comment:
The notification of copying block ref is gone in 0.9.2, is this a feature or bug?